src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -135,6 +135,12 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('set_value', options_list=['--set'], help="Value in 'name[=value]' format. Multiples supported by passing --set multiple times.", action='append', validator=validate_set)
         c.argument('set_secret', help="Secret value in '--set name[=value]' format. Multiples supported by passing --set multiple times.", action='append', validator=validate_set_secret)
 
+    with self.argument_context('acr pack') as c:
+        c.argument('registry_name', options_list=['--registry', '-r'])
+        c.argument('image_name', options_list=['--image', '-t'], help="The name and tag of the image using the format: '-t repo/image:tag'.")
+        c.argument('builder', options_list=['--builder', '-b'], help="The name and tag of a Buildpack builder image.")
+        c.positional('source_location', help="The local source code directory path (e.g., './src') or the URL to a git repository (e.g., 'https://github.com/Azure-Samples/acr-build-helloworld-node.git') or a remote tarball (e.g., 'http://server/context.tar.gz'). If '/dev/null' is specified, the value will be set to None and ignored.", completer=FilesCompleter())
```

Comment:
>If '/dev/null' is specified, the value will be set to None and ignored. [](start = 263, length = 71)

For buildpack scenario, do you support empty build context? I assume you always need to pass the actual source location, either local source folder or git url?

Reply:
I copied this line blindly from the block above - I fixed the documentation to match the actual usage.